guides/doc-Project_API/topics/con_api-cheat-sheet.adoc

```diff
@@ -0,0 +1,23 @@
+[id="api-cheat-sheet"]
+= API cheat sheet
+
+You can review the following examples of how to use the {ProjectName} API to perform various tasks.
+You can use the API on {ProjectServer} via HTTPS on port 443 or on {SmartProxyServer} via HTTPS on port 8443.
+
+You can address these different port requirements within the script itself.
+For example, in Ruby, you can specify the {Project} and {SmartProxy} URLs as follows:
+
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+url = 'https://_{foreman-example-com}_/api/v2/'
+{smart-proxy-context}_url = 'https://_{smartproxy-example-com}_:8443/api/v2/'
+katello_url = 'https://_{foreman-example-com}_/katello/api/v2/'
+----
+
+For the host that is subscribed to {ProjectServer} or {SmartProxyServer}, you can determine the correct port required to access the API from the */etc/rhsm/rhsm.conf* file, in the port entry of the `[server]` section.
```

Comment:
```suggestion
For the host that is registered to {ProjectServer} or {SmartProxyServer}, you can determine the correct port required to access the API from the `/etc/rhsm/rhsm.conf` file, in the port entry of the `[server]` section.
```

Reply:
personal note: I want to check if this is true for all Client OS; tested on AlmaLinux, Ubuntu, and SLES; file belongs to subscription-manager. :heavy_check_mark: